lib/sles4sap_publiccloud_basetest.pm

```diff
@@ -56,6 +96,57 @@ sub import_context {
     );
 }
 
+=head2 set_cli_ssh_opts
+
+    $self->set_cli_ssh_opts();
+    $self->set_cli_ssh_opts('');
+    $self->set_cli_ssh_opts("-4 -o LogLevel=ERROR -E $logfile");
+
+Set command line SSH options in the instance stored in C<$self-E<gt>{my_instance}>. It takes
+as an argument a string with the options in a manner that would be understood by B<ssh>, and
+if no argument is provided, uses the following defaults:
+
+=over
+
+=item *
+
+C<-E /var/tmp/ssh_sut.log>: save logging to B</var/tmp/ssh_sut.log>.
+
+=item *
+
+C<-F none>: do not use SSH configuration files.
+
+=item *
+
+C<-o LogLevel=DEBUG3>: set log level to B<DEBUG3>.
+
+=item *
+
+C<-o PasswordAutentication=no>: do not allow authentication via password.
+
+=item *
+
+C<-i 'get_ssh_private_key_path()'>: use the generated private SSH key
+
+=back
+
+B<Note>: if the method receives an empty string, no SSH options will be set.
+
+=cut
+
+sub set_cli_ssh_opts {
+    my ($self, $ssh_opts) = @_;
+    croak("Expected \$self->{my_instance} is not defined. Check module Description for details")
+      unless $self->{my_instance};
+    $ssh_opts //= join(' ',
+        '-E', '/var/tmp/ssh_sut.log',
+        '-F', 'none',
```

Comment:
we can add a command to explain we use `-F none`

Reply:
On top of https://os-autoinst.github.io/os-autoinst-distri-opensuse/sles4sap_publiccloud_basetest.html#set_cli_ssh_opts ??

Reply:
not needed, no